data/names/ru.json

```diff
@@ -3031,22 +3033,27 @@
       "Айверсон",
       "Айви",
       "Айерс",
-      "Айзекс",
       "Айзек",
+      "Айзекс",
       "Айкен",
       "Айрлэнд",
       "Акерман",
+      "Акерс",
       "Акино",
       "Акинс",
       "Акоста",
       "Акуна",
+      "Аланиз",
       "Аланис",
       "Аларкон",
       "Алвес",
       "Александер",
+      "Александр",
       "Алеман",
       "Али",
       "Аллен",
+      "Аллея",
+      "алмаз",
```

Comment:
This is just a translation of the common noun "diamond" (as in the mineral). There *is* a name that's spelled exactly like this, but it would start with a capital letter.

Reply:
You could also transliterate as Диамант